lib/msal-browser/src/interaction_handler/SilentHandler.ts

```diff
@@ -145,9 +139,11 @@ export class SilentHandler extends InteractionHandler {
      * Creates a new hidden iframe or gets an existing one for silent token renewal.
      * @ignore
      */
-    private createHiddenIframe(): HTMLIFrameElement {
+    private createHiddenIframe(urlNavigate: string): HTMLIFrameElement {
         const authFrame = document.createElement("iframe");
 
+        authFrame.className = "silentTokenRenewalIframe";
+        authFrame.src = urlNavigate;
```

Comment:
Is there a reason why we did not do this before? @tnorling  any browser policy issues we need to worry about?

Reply:
There was a limitation in some older browsers, namely IE11, that affected iframe creation. To workaround we allowed applications to configure whether the src attribute should be assigned synchronously or asynchronously - we can likely remove that configuration in v4 but we'll need to keep the existing model in v3 to avoid a change in behavior here

Reply:
Noted, thanks!

Reply:
That is what I was wondering. @lalimasharda  let us stick to `className` for now to unblock Dime.